edi_exchange_deduplicate_oca/models/edi_exchange_record.py

```diff
@@ -39,7 +39,7 @@ def create(self, vals_list):
 
     def _edi_get_duplicates(self, count=None):
```

Comment:
Should we not set the boolean `False` for the count parameter instead of `None` ?

Reply:
Well, we only use it as a check for deciding whether we're gonna use `search()` or `search_count()`, so as long as it can be evaluated as a boolean, it doesn't make much difference, any `False`-ish default value should be fine :smile:

Reply:
Indeed but I find it a bit misleading and in the search method [back in V16,](https://github.com/odoo/odoo/blob/16.0/odoo/models.py#L1511C5-L1511C77) the default value was `False`

But this is just design :smile: